kernel/mutex.c

```diff
@@ -26,6 +26,7 @@
  * unexpected priority levels (too high, or too low).
  */
 
+// new comment
 #include <kernel.h>
 #include <kernel_structs.h>
```

Comment:
CHECKPATCH ERROR:
 open brace '{' following function declarations go on the next line